MapboxSpeech/MapboxSpeech.swift

```diff
@@ -58,19 +58,15 @@ let userAgent: String = {
 open class SpeechSynthesizer: NSObject {
     
     public typealias CompletionHandler = (_ data: Data?, _ error: NSError?) -> Void
-    
-    // MARK: Creating a Speech Object
-    
-    @objc(sharedSpeech)
-    open static let shared = SpeechSynthesizer(accessToken: nil)
```

Comment:
Instead of removing this initializer, remove `accessToken: nil`. I think that makes the intent clearer.